.circleci/config.yml

```diff
@@ -39,4 +39,4 @@ workflows:
           matrix:
             parameters:
               # https://github.com/CircleCI-Public/cimg-ruby
-              ruby-version: ["2.7", "3.0", "3.1"]
+              ruby-version: ["3.1"]
```

Comment:
Should we also add `3.2` and `3.3` to the ruby matrix?

Reply:
Good idea, added in [2c8f5a3](https://github.com/ezcater/ezcater_rubocop/pull/130/commits/2c8f5a39bd7f725038f8fc7f6ecbec9a17587dd8)